docs/references/expo/expo-impersonation.mdx

```diff
@@ -0,0 +1,509 @@
+1. You must configure your Clerk instance to enable **Email and Password** authentication:
+    1. Navigate to the Clerk Dashboard in the navigation sidebar, go to **User & Authentication > [Email, Phone, Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)**.
+    1. Ensure that **Email address** is required, and **Username** is not required.
+    1. Ensure that **Password** is required.
+1. Install the following dependencies:
+    <CodeBlockTabs options={["npm", "yarn", "pnpm"]}>
+    ```bash {{ filename: "terminal" }}
+    npm i expo-linking && npm i @clerk/types
+    ```
+
+    ```bash {{ filename: "terminal" }}
+    yarn add expo-linking && yarn add @clerk/types
+    ```
+
+    ```bash {{ filename: "terminal" }}
+    pnpm add expo-linking && pnpm add @clerk/types
+    ```
+    </CodeBlockTabs>
+1. Now that Clerk is configured, you'll need to [deploy to Vercel](https://docs.expo.dev/router/reference/api-routes/#vercel).
+
+<Steps>
+
+### Create a `(dashboard)` route
+
+> [!WARNING]
+> This is a basic example that foregoes best practices for the sake of brevity and shouldn't be copied one-for-one into a _production_ Expo application. For example, you should build impersonation into an admin-only dashboard that **only authorized users** can access.
+
+For this example, create a basic dashboard for impersonating users.
+
+The person visiting this dashboard should not be able to see it unless signed in. Start by creating the `(dashboard)` route group and adding the following code to the layout:
+
+```tsx {{ filename: "app/(dashboard)/_layout.tsx" }}
+import { Redirect, Stack } from "expo-router";
+import { useAuth } from "@clerk/clerk-expo";
+
+export default function GuestLayout() {
+  const { isSignedIn } = useAuth();
+
+  if (!isSignedIn) {
+    return <Redirect href={"/dashboard"} />;
+  }
+
+  return <Stack />;
+}
+```
+
+### Create a `(dashboard)` page
+
+Create the main page of the dashboard, which will hold most of your impersonation code. This example contains the skeleton of the final result, which will be fleshed out with each step in this guide.
+
+> To skip this process, see [the full example](#full-example) at the end of this guide.
+
+```tsx {{ filename: "app/(dashboard)/index.tsx" }}
+import React from "react";
+import { Button, StyleSheet, Text, TouchableOpacity, View } from "react-native";
+import { Link, useRouter } from "expo-router";
+import { useAuth, useUser, useSignIn, useSessionList } from "@clerk/clerk-expo";
+import { UserDataJSON } from "@clerk/types";
+import * as Linking from "expo-linking";
+import useImpersonation from "@/hooks/useImpersonation";
+
+// this hook will return all of the user data for our impersonator, using an API route
```

Comment:
Should these still be in this file/section? I'm pretty sure that lines 9-13 should be removed in favour of the imports.